tests/integration/cmd_project_test.go

```diff
@@ -0,0 +1,51 @@
+package integration
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo project command tests", func() {
```

Comment:
+100 for new test file. Later it will be easy to improve with all possible project command scenarios.